sdk/synapse/synapse-artifacts/karma.conf.js

```diff
@@ -4,14 +4,11 @@
 // https://github.com/karma-runner/karma-chrome-launcher
 process.env.CHROME_BIN = require("puppeteer").executablePath();
```

Comment:
For synapse artifacts, node tests are fine.

However, browser tests are failing... error stack doesn't seem recorder related.

And the live tests are failing as well in the browser.



So, I believe the problem is coming from core-client and not the recorder/.



![image](https://user-images.githubusercontent.com/10452642/151862563-b576c0c0-e022-455f-8d94-edb56911b1e1.png)